Packs/ShiftManagement/Playbooks/Shift_handover.yml

```diff
@@ -0,0 +1,1663 @@
+  "5":
+    id: "5"
+    taskid: d8ec2809-c7b5-45e6-8596-68c55e30392f
+    type: title
+    task:
+      id: d8ec2809-c7b5-45e6-8596-68c55e30392f
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 2490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "6":
+    id: "6"
+    taskid: de8073a7-2c08-421a-87d3-6fcb7f70c362
+    type: regular
+    task:
+      id: de8073a7-2c08-421a-87d3-6fcb7f70c362
+      version: -1
+      name: Find similar incidents
+      description: |-
+        Finds similar incidents by common incident keys, labels, custom fields, or context keys.
+        It's highly recommended to use incident keys if possible (e.g., "type" for the same incident type).
+        For best performance, it's recommended to avoid using context keys if possible (for example, if the value also appears in a label key, use label).
+      scriptName: FindSimilarIncidents
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      filterQuery: {}
+      hoursBack: {}
+      ignoreClosedIncidents: {}
+      incidentFieldsAppliedCondition: {}
+      maxNumberOfIncidents: {}
+      maxResults: {}
+      similarContextKeys: {}
+      similarCustomFields: {}
+      similarIncidentFields:
+        simple: type
+      similarIncidentKeys: {}
+      similarLabelsKeys: {}
+      skipMissingValues: {}
+      timeField: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": -475
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "7":
+    id: "7"
+    taskid: 94a1ff3b-36df-4ab4-8b9a-71d9a7a77991
+    type: condition
+    task:
+      id: 94a1ff3b-36df-4ab4-8b9a-71d9a7a77991
+      version: -1
+      name: Found similar incidents?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "18"
+      "yes":
+      - "8"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: similarIncident
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": -250
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "8":
+    id: "8"
+    taskid: 9c8c9a00-b552-4178-8ea7-2fdf55284b23
+    type: regular
+    task:
+      id: 9c8c9a00-b552-4178-8ea7-2fdf55284b23
+      version: -1
+      name: Close previous change shift incident
+      description: commands.local.cmd.close.inv
```

Comment:
@ShirleyDenkberg this is a server default description and not the correct description.